docker-compose.yml

```diff
@@ -1,34 +1,21 @@
 services:
-  frontend:
+  upgrade-app:
     build:
       context: .
-      dockerfile: ./frontend/dev.Dockerfile
-    image: upgrade-frontend:latest
-    user: "node"
-    environment:
-      - NODE_ENV=development
-    volumes:
-      - ./frontend:/usr/src/app/frontend
-    networks:
-      - localdev
-    ports:
-      - "4200:4200"
-  
-  backend:
-    build:
-      context: .
-      dockerfile: ./backend/dev.Dockerfile
-    image: upgrade-backend:latest
+      dockerfile: ./dev.Dockerfile
+    image: upgrade-app:latest
     user: "node"
     env_file:
       - ./backend/packages/Upgrade/.env.docker.local
     environment:
       - NODE_ENV=development
```

Comment:
Not sure if we should keep this as development or production?

Reply:
I feel we are using this docker for development, as it was in previous dockerfile as well. @danoswaltCL @kawcl Should we have a production version for the docker image that gets pushed automatically to dockerhub using github actions?

Reply:
Maybe? Originally the setup was only meant to be a dockerized local setup. I like the idea of having a production-ready docker image that we pop into dockerhub, and maybe they can be the same basic image but I'm probably not the right person to review it, I'd like to run these ideas by our SRE guy @kcalvo-cli and @kawcl when we get back after the holidays before we merge this in. We definitely talked about that kind of thing when we were helping another team implement UpGrade on their prod env.